src/iexamine.cpp

```diff
@@ -56,39 +56,34 @@ void iexamine::gaspump(player *p, map *m, int examx, int examy)
     add_msg(m_info, _("Out of order."));
 }
 
-void iexamine::atm(player *p, map *m, int examx, int examy)
+void iexamine::atm(player *const p, map *, int , int )
 {
-    (void)m; //unused
```

Comment:
You can also remove their name from the parameter list, but neither are *necessary* -- they just produce a warning on most compilers. With the warning settings DDA uses though, those warnings will be flagged as errors.